### PR TITLE
Log slowest CI tests

### PR DESCRIPTION
**Pull request by tamird**
As discussed in #1500.

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -49,6 +49,10 @@ test:
     - "[ -f ${CIRCLE_ARTIFACTS}/testrace.log ] && docker run -i cockroachdb/cockroach-dev shell /go/bin/go2xunit < ${CIRCLE_ARTIFACTS}/testrace.log > ${CIRCLE_TEST_REPORTS}/race/testrace.xml"
     - |
       find "${CIRCLE_ARTIFACTS}" -name '*.log' -type f -exec \
+        grep -F ': Test' | sed -E 's/(--- PASS: |\(|\))//g' | awk '{ print $2, $1 }' | sort -rn | head -n 10 \
+        > "${CIRCLE_ARTIFACTS}/slow.txt"
```
Comment (tschottdorf) [inline circle.yml]: that's only one output overwritten for multiple inputs, or am I missing something?

File: circle.yml [hunk 2]
```diff
@@ -49,6 +49,10 @@ test:
...
+    - |
+      find "${CIRCLE_ARTIFACTS}" -name '*.log' -type f -exec \
         grep -B 5 -A 10 -E '^\-{0,3} *FAIL|^panic|^[Gg]oroutine \d+|(read|write) by.*goroutine|DATA RACE' {} ';' \
```
Comment (tschottdorf) [inline circle.yml]: ditto

Comment (tschottdorf) [inline circle.yml]: (even in the original... ;) )

Comment (tamird) [inline circle.yml]: fixed :)

File: circle.yml [hunk 3]
```diff
@@ -49,8 +49,12 @@ test:
     - "[ -f ${CIRCLE_ARTIFACTS}/testrace.log ] && docker run -i cockroachdb/cockroach-dev shell /go/bin/go2xunit < ${CIRCLE_ARTIFACTS}/testrace.log > ${CIRCLE_TEST_REPORTS}/race/testrace.xml"
     - |
       find "${CIRCLE_ARTIFACTS}" -name '*.log' -type f -exec \
+        grep -F ': Test' | sed -E 's/(--- PASS: |\(|\))//g' | awk '{ print $2, $1 }' | sort -rn | head -n 10 \
+        >> "${CIRCLE_ARTIFACTS}/slow.txt"
+    - |
+      find "${CIRCLE_ARTIFACTS}" -name '*.log' -type f -exec \
```
Comment (tschottdorf) [inline circle.yml]: stable order would be better, but this'll probably be more than enough for the casual look.

**Comment by tschottdorf** [comment]
LGTM (if it works in the running build)

**Comment by tamird** [comment]
Well this produces duplicated output, but otherwise works.

**Pull request merged**